cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -0,0 +1,268 @@
+describe('Comment on articles', () => {
```

Comment:
I thought about this, but because we now switch from plain textarea to combobox textarea, I think in each test asserting we're in the correct textarea is sufficient? Otherwise we're just testing standard textarea functionality 🙃 

It's a bit of a pity the Cypress tests in this case have to be quite coupled to the implementation - at each step we need to get a new handle to the currently presented textarea